tutorials/algorithms/09_IQPE.ipynb

```diff
@@ -524,19 +557,21 @@
   },
   {
    "cell_type": "markdown",
+   "id": "d5b1356c",
    "metadata": {},
    "source": [
-    "Let's execute the circuit with the `qasm_simulator`, the simulator without noise that run locally."
+    "Let's sample the circuit with Qiskit Aer's `Sampler` primitive, a local simulator without noise that runs locally."
```

Comment:
Is there a Terra implementation of `Sampler`? Aer is going to be removed from the meta package so whenever we can we should use terra only.

Reply:
There is, but it doesn't support the circuits constructed here, which re-use classical bits for multiple measurements.